README.md

```diff
@@ -359,34 +359,47 @@ Later, when someone tries to validate the signature, they can check that the cer
+The SBOM includes the dependencies of an artifact and may include licensing information about the artifact and its dependencies.
+While an SBOM can be useful to enforce internal licensing standards, it's also valuable for vulnerability analysis; this is especially true when there's a need to retroactively look at software running in production and determine if it's vulnerable to a new attack.
+If an organization already has SBOMs for the software it's running, it becomes much easier to tell if there's an impact from a new vulnerability.
+
+There are several standard formats, like [SPDX](https://spdx.dev/) from the Linux Foundation and [CycloneDX](https://cyclonedx.org/) from OWASP.
+Fortunately, there are a number of tools for generating and working with SBOMs and most tools understand multiple standards. In this demonstration, we're using SPDX as the format and [Syft](https://github.com/anchore/syft) as the tool to generate SBOMs.
+
 ## Workflows
 
 The demo makes use of several reusable workflows defined in [`liatrio/gh-trusted-builds-workflows`](https://github.com/liatrio/gh-trusted-builds-workflows). 
 Each workflow is owned by either the platform or security teams.
 
-![Workflow run](./assets/workflows.png)
+![Workflow run](https://raw.githubusercontent.com/liatrio/gh-trusted-builds-app/main/assets/workflows.png)
 
 ### Platform: Build & Push
 
 The platform team's `build-and-push` workflow is split into several jobs:
-- `source-attestations`
+- `detect-workflow`
 - `build`
 - `push`
 - `sign`
+- `source-attestations`
 - `provenance`
+- `sbom`
 
 The `build` job uses Docker to build a container image, but doesn't push it to a container registry. 
-Instead, the job outputs a tar file that can be pushed later. The reason to split build and push is that the build step is executing potentially untrusted code from the application team, and we don't want that job to be able to request the id token that's used for signing or to try to push a malicious image. So the `build` job only has the permissions it needs in order to checkout the repo:
+Instead, the job outputs a tar file that will be pushed by the next job. 
+The reason to split build and push is that the build step is executing untrusted code from the application team; consequently, we don't want that job to be able to request the id token that's used for signing or to try to push a malicious image.
+So the `build` job only has the permissions it needs in order to checkout the repo:
 
 ```yaml
 jobs:
   build:
     permissions:
-      id-token: none
       contents: read
 ```
 
-Next up is the `push` job, which loads the tar file from the `build` job and pushes it to GitHub Container Registry (GHCR). Like the build job, it only has the permissions it needs:
+Next up is the push job, which loads the tar file from the build job and pushes it to GitHub Container Registry (GHCR). Like the build job, it only has the permissions it needs to checkout the repository and write to the registry:
```

Comment:
I removed the checkout step from push, it didn't have permissions for content read either.